lib/compute/agent-nodes.ts

```diff
@@ -152,10 +152,10 @@ export class AgentNodes {
         workerLabelString: 'Jenkins-Agent-Windows2019-X64-C54xlarge-Single-Host',
         instanceType: 'C54xlarge',
         remoteUser: 'Administrator',
-        maxTotalUses: -1,
-        minimumNumberOfSpareInstances: 1,
+        maxTotalUses: 10,
```

Comment:
Why restrict to 10?

Reply:
A bug that I have yet to have ways to fix it:
https://github.com/opensearch-project/opensearch-build/issues/2306#issuecomment-1218505002
https://github.com/opensearch-project/opensearch-build/issues/2306#issuecomment-1219955242